knowledge-base/adr/0020-generator-options.md

```diff
@@ -37,26 +37,35 @@ We should discuss the details of those defaults and make a concious decision on
 | **Options in OData, but not (yet in OpenAPI) generator** |
-| `--forceOverwrite`                                       | `--overwrite`                    | Overwrite files even it they exist, default false.                                                | _same_                                                                                                                                                       |
-| `--generateTypedocJson`                                  | `--typedocJson`                  | Writes a default typedoc.json, default true.                                                      | Remove/deprecate, use `files` instead.                                                                                                                       |
+| `--forceOverwrite`                                       | `--overwrite`                    | Overwrite files even it they exist, default false.                                                | _same_                                                                                                                                                                                                         |
+| `--generateTypedocJson`                                  | `--typedocJson`                  | Writes a default typedoc.json, default true.                                                      | Remove/deprecate, use `files` instead.                                                                                                                                                                         |
 | **Currently hidden options**                             |
-| `--additionalFiles`                                      | `--include`                      | Copy additional files, identified by glob. Hidden.                                                | _same_ Expose it.                                                                                                                                            |
-| `--writeReadme`                                          | `--readme`                       | Writes a default README.md, default false.                                                        | _same_                                                                                                                                                       |
+| `--additionalFiles`                                      | `--include`                      | Copy additional files, identified by glob. Hidden.                                                | _same_ Expose it.                                                                                                                                                                                              |
```

Comment:
was the name not supposed to change to `includes` or `files` in the future?

Reply:
Yes, it is `include` (you have to look in the second column for the new names).